DemoApp/CreateChatViewController.swift

```diff
@@ -285,7 +285,7 @@ extension CreateChatViewController: UITableViewDelegate, UITableViewDataSource {
         let client = searchController.client
         composerView.controller = try? client
             .channelController(
-                createDirectMessageChannelWith: selectedUserIds.union([client.currentUserId]),
+                createDirectMessageChannelWith: selectedUserIds.union(client.currentUserId.map { [$0] } ?? []),
```

Comment:
We should handle this properly and show to our user what to do in these cases. When the current user is `nil` it's clear something is wrong and it doesn't make sense to use the fallback. The request will fail aynway.

Reply:
Shouldn't it be the `LLC` responsibility to include `currentUserID` automatically when `createDirectMessageChannel` API is used? 🤔

Reply:
@evsaev right. It possible we don't need to include the current user id here, too, and the backend will add it automatically.